### PR TITLE
Added unit tests for setMinFileSize method in FileFilter

### LINKED ISSUE #12: Write Unit Tests for `setMinFileSize` Method
- **Description**: Write unit tests to validate the behavior of the `setMinFileSize` method. Ensure that it correctly sets the minimum file size and that it can handle invalid inputs.
   - **Resources**: Refer to the [pytest getting started guide](https://docs.pytest.org/en/stable/getting-started.html) for setting up tests.
   - **Path**: `twinTrim/dataStructures/fileFilter.py`
   - **Note**: Ensure to maintain the file structure as per the existing files and tests, and add these tests.

### PR DESCRIPTION
**Pull request by smartfellow1234**
## Description

Added unit tests for the setMinFileSize method in the FileFilter class to validate that it correctly sets the minimum file size. The tests also ensure that the method handles invalid inputs gracefully and behaves as expected when various sizes are provided.

- Fixes #12 

## Type of change

- [X] Tests update

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] I have maintained a clean commit history by using the necessary Git commands
- [X] I have checked that my code does not cause any merge conflicts

## Screenshots (if applicable)
image

### CODE REVIEW

**Comment by techy4shri** [comment]
@smartfellow1234 take a look at what I changed in your description comment and keep that in mind, it will help you nect time you raise a PR.

**Comment by techy4shri** [comment]
@smartfellow1234 wait for review from Karthik, the PA of this repository for now.

**Review by Kota-Karthik — APPROVED** [review]
@smartfellow1234 
everything LGTM!

**Pull request merged**
